python-backend/app/api/now_applications/models/state_of_land.py

```diff
@@ -6,15 +6,16 @@
 from app.extensions import db
 
 
-class StateOfLand(AuditMixin, Base):
```

Comment:
I presume that this AuditMixin was removed because of `StateOfLand`'s unique 1-to-1 relationship with `NOWApplication`? Will updates to `StateOfLand` specific fields be reflected in the Audit columns of the parent table?

Reply:
Yes, they are removed because logically they are the same entity as the parent. When state of land details are updated, the parent audit columns will reflect that.